packages/base-map/src/index.js

```diff
@@ -47,6 +49,15 @@ L.Evented.include({
  * The base OpenTripPlanner map on which everything else is rendered.
  */
 class BaseMap extends Component {
+  overlays = [];
+
+  constructor() {
+    super();
+    this.handleOverlayAdded = this.handleOverlayAdded.bind(this);
+    this.handleOverlayRemoved = this.handleOverlayRemoved.bind(this);
+    this.handleViewportChanged = this.handleViewportChanged.bind(this);
```

Comment:
These bindings can be done by rewriting the class methods as follows:

```js
handleOverlayAdded = e => {
  ...
}
```

Reply:
Good catch, the handlers were already written this way...